PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -196,6 +199,7 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
         mvaFall17V1noIso = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Fall17NoIsoV1Values"),
         mvaFall17V2Iso = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Fall17IsoV2Values"),
         mvaFall17V2noIso = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Fall17NoIsoV2Values"),
```

Comment:
We should rename these ID to be generic.
In Run3 we will soon move out of the "Fall17V2" training .
So in the nano we should have one variable generica name and will be dinamically filled with the right ID for the Run2 ("Fall17V2" training) and Run3 (the one to come) 
mvaFall17V2Iso --> mvaIso
mvaFall17V2noIso -->  mvanoIso
The docs will point to the right one.

At the analysis level this will help the combined analysis of the Run2 and Run3 dataset, that is our main goal for next Run2 nano production.

@lfinco @swagata87 @rgoldouz

Reply:
Hi @mariadalfonso , 
yes I agree. 
@asculac you can implement it in this PR. Or if you feel this is out of scope of this PR, then one of us can do it in a follow-up PR. But indeed, the naming of mva egamma ID branches could be improved. (The cutbased ID branch names already looks good.)

Reply:
It looks like a very good idea; I would then suggest  to agree on consistent naming;
mvaIso
mvanoIso (or better mvaNoIso)?
mvaHZZIso (instead  of mvaHZZIdIsoValue)

Reply:
@swagata87 sure, I will add it to this PR it is not a problem
@mariadalfonso thank you for pointing everything out, I will check it and fix it